doc/material-controls-extensions.md

```diff
@@ -48,90 +48,3 @@ It's control specific and for now, you can only use it with the ToggleButton con
     </um:ControlExtensions.AlternateContent>
 </ToggleButton>
 ```
-
-## Elevation
-
-This feature allows to set the level of elevation to depict on the supported control.
-
-Setting the elevation on [supported controls](#supported-controls) can result in changes to both the shadow and the [surface tint](#surface-tint).
-
-[Material Design Elevation Guidance](https://m3.material.io/styles/elevation/overview)

```

Comment:
We didn't remove the Elevation attached property in the ControlExtensions, right? So we should just remove the mention of surface tint in this section